apps/authentication/mfa/face.py

```diff
@@ -30,10 +30,11 @@ def is_active(self):
 
     @staticmethod
     def global_enabled():
-        return settings.XPACK_LICENSE_IS_VALID \
-            and LicenseEditionChoices.ULTIMATE == \
-            LicenseEditionChoices.from_key(settings.XPACK_LICENSE_EDITION) \
-            and settings.FACE_RECOGNITION_ENABLED
+        return (
+            settings.XPACK_LICENSE_IS_VALID and
+            settings.XPACK_LICENSE_EDITION_ULTIMATE and
+            settings.FACE_RECOGNITION_ENABLED
+        )
 
     def get_enable_url(self) -> str:
         return '/ui/#/profile/index'
```

Comment:
The code has no significant changes between the two provided lines of code.

There aren't any apparent inconsistencies that would cause trouble during this review. However, there could be some unnecessary redundancy in the second line, but it seems to serve its purpose well:

```python
@staticmethod
def global_enabled():
   return (
       settings.XPACK_LICENSE_IS_VALID and 
       settings.XPACK_LICENSE_EDITION_ULTIMATE if (settings.FACE_RECOGNITION_ENABLED else True)
   )
```

For further optimization purposes, consider breaking down complex boolean variables like `settings.XPACK_LICENSE_EDITION` into separate constants for readability and maintainability. The current implementation uses nested ternary structures which can make the logic more cryptic and harder to understand or debug in future when code changes happen.

In summary, the provided code doesn't require any major adjustments, however, introducing better variable handling and separation is beneficial. Here's an improved version with simplified string operations:

```python
class UserConfigurator:
    LICENSE_EDITIONS = {LicenseEditionChoices.LICENSE_KEY_ULTIMATE.lower(): LicenseEditionChoices, LicenseEditionChoices}

    @property
    def face_recognition_enabled(self):
        # Use the enum value directly since setting a new enum member here requires modifying existing data structure
        return self.LESIONS[Settings.XPACK_LICENSE_EDITION.value]

    @face_recognition_enabled.setter
    def face_recognition_enabled(self, enabled: bool | None):
        pass

    def validate_xpack_license(self) -> bool:
        return settings.XPACK_LICENSE_IS_VALID and self.face_recognition_enabled
    ```
I hope these suggestions are helpful!